Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:bookworm-slim as builder
+FROM ubuntu:24.04 as builder
```

Comment:
We should probably stick to the initial choice of using a debian image (especially if it's smaller) if using trixie is fine.

Reply:
OK I've changed back to debian trixie